### DISCUSSION TITLE
Error at the end of print

### DISCUSSION THREAD

**Discussion by tjhayko** [bassamanator/Sovol-SV06-firmware#188] [General]
I'm getting a error at the end of my print.

error: Move out of range: 226.075 485.788 35.100 [23923.196]

I only see it the KlipperScreen.log at this point.

I have captured the whole printer_data folder, as well as the .gcode for the part that was printing.

I'm using Cura for slicing.

What are the steps I can take to figure out what is wrong?

The odd thing is that my part seemed to print to the end fine, or at least close enough that it's usable.

Thanks in advance!

**Comment by bassamanator** [comment]
You've adjusted the slicer as per the adjust your slicer section?

**Comment by tjhayko** [comment]
I had, but I restored a Cura config that had set it back to what it was before I started using Klipper.

Thanks for the pointer!

**Comment by bassamanator** [comment]
Welcome.

Please star the repo and close the discussion when ready. Thanks.
